packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,25 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    #[cfg_attr(feature = "clap", clap(global = true, long, default_value_t = 0))]
```

Comment:
Can we set `#[cfg_attr]` down to the function, so the comment is above it? This makes it easier to read.

Reply:
Addressed in https://github.com/hashintel/hash/pull/446/commits/5967afd1d09d3af82dcc2a07620d79e23de506ae